### PR TITLE
[YUNIKORN-2870] Release notes for 1.6.0

### PR DESCRIPTION
**Pull request by pbacsko**
### What is this PR for?
Release announcement docs.

### What type of PR is it?
* [ ] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [x] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-2870

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

File: src/pages/release-announce/1.6.0.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+Release manager: Peter Bacsko
+
+Release date: 2024-09-17
+
+## Highlights
+
+### Unification of Ask and Allocation
+
+YuniKorn 1.6.0 got simplified on a code level - we no longer have "Ask" objects in
```
Comment (craigcondit) [inline src/pages/release-announce/1.6.0.md:41]: nit: "got" -> "has been"

File: src/pages/release-announce/1.6.0.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+### Support for OIDC credentials
+
+In [YUNIKORN-2281](https://issues.apache.org/jira/browse/YUNIKORN-2281) we added support
+for OIDC user names. Previously, these got rejected in the admission controller
+because certain characters were treated as illegal. 
+
+### Improved code quality
+
+In [YUNIKORN-182](https://issues.apache.org/jira/browse/YUNIKORN-182) we improved the
+code quality of Yunikorn so that the Go linterns no longer complain about potential
```
Comment (craigcondit) [inline src/pages/release-announce/1.6.0.md:68]: spelling: "linterns" -> "linters"

File: src/pages/release-announce/1.6.0.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+coding problems.
+
+Coverage of various parts of the code got significantly enhanced. See the [JIRA filter](https://issues.apache.org/jira/issues/?filter=12353613)
```
Comment (craigcondit) [inline src/pages/release-announce/1.6.0.md:71]: nit: "got" -> "was"

File: src/pages/release-announce/1.6.0.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+Before YuniKorn 1.6.0, labels and annotations were not used in a consistent way.
+Certain metadata existed either as a label or annotation. If it existed for both,
+the precedence was not clear. If both were defined with different values, no error
+was reported. This got fixed in [YUNIKORN-2501](https://issues.apache.org/jira/browse/YUNIKORN-2501).
+
+### Support for OIDC credentials
+
+In [YUNIKORN-2281](https://issues.apache.org/jira/browse/YUNIKORN-2281) we added support
+for OIDC user names. Previously, these got rejected in the admission controller
```
Comment (craigcondit) [inline src/pages/release-announce/1.6.0.md:62]: nit: "got" -> "were"

**Review by craigcondit — CHANGES_REQUESTED** [review]
Minor grammer / spelling issues.

**Review by craigcondit — APPROVED** [review]
+1 LGTM.

File: src/pages/release-announce/1.6.0.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+orchestration frameworks like Kubernetes for on-prem and on-cloud use cases.
+
+## Overview
+The Apache YuniKorn community has resolved 316 [JIRAs](https://issues.apache.org/jira/issues/?filter=12352202) in this release.
+
+Release manager: Peter Bacsko
+
+Release date: 2024-09-17
+
+## Highlights
```
Comment (wilfred-s) [inline src/pages/release-announce/1.6.0.md:37]: Need to add the new K8s releases supported

**Review by wilfred-s — CHANGES_REQUESTED** [review]

**Review by brandboat — APPROVED** [review]
LGTM, thanks everyone for the great work :tada:

**Review by craigcondit — APPROVED** [review]
+1 LGTM.

**Pull request closed without merging**